dblookupext/factory/historyRepositoryFactory_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/ElrondNetwork/elrond-go/config"
 	"github.com/ElrondNetwork/elrond-go/dataRetriever"
 	"github.com/ElrondNetwork/elrond-go/dblookupext/factory"
+	"github.com/ElrondNetwork/elrond-go/process"
+	mock2 "github.com/ElrondNetwork/elrond-go/process/mock"
```

Comment:
mock2 ?

Reply:
Consider replacing `mock2` with `processorMock` alias

Reply:
Renamed it to `processMock`